File: 01_Language/01_Functions/python/date_diff.py
# coding: utf-8

import datetime

d1 = datetime.datetime(2009, 10, 11)
d2 = datetime.datetime(2009, 10, 13)
print((d2 - d1).days)
print((d2 - d1).seconds)
print((d2 - d1).total_seconds())

File: 01_Language/01_Functions/php/date_diff.php
<?php

$datetime1 = new DateTime('2009-10-11');
$datetime2 = new DateTime('2009-10-13');
$interval = $datetime1->diff($datetime2);
echo $interval->format('%R%a days');

$datetime1 = date_create('2009-10-11');
$datetime2 = date_create('2009-10-13');
$interval = date_diff($datetime1, $datetime2);
echo $interval->format('%R%a days');
